coral/src/app/layout/Layout.tsx

```diff
@@ -28,7 +31,7 @@ function Layout({ children }: { children: ReactNode }) {
           <Header />
         </GridItem>
         <GridItem colStart={"1"} colEnd={"2"} rowStart={"2"}>
-          <MainNavigation />
+          <MainNavigation authUser={authUser} />
```

Comment:
Unsure about passing `authUser` as a prop :o  Is there a reason to not call `useAuthContext` in `mainNavigation` directly?

Reply:
Yes, that would make more sense now! 
I had it in there before and only let the Layout render anything if there's an auth user, because before we actually rendered a lot of content already (nothing from APIs ofc) before we redirected the not-authenticated users. But now the AuthProvider is taking care of that, so we can move it there!